text/0000-project-groups.md

```diff
@@ -0,0 +1,391 @@
+  RFCs and doing design work.
+- Each project group typically has:
+  - A charter outlining the group's scope and goals.
+  - Appointed shepherds and team liasons.
+  - An associated repository.
+  - Regular meetings.
+  - Dedicated streams on Discord/Zulip/etc.
+- Define working groups to refer to the "domain working groups" that are created
+  to explore particular domains, such as embedded, CLI, etc.
+  - They have a charter and defined leads but operate more independently from
+    the Rust teams.
+- Define community group as the term for groups not formally affiliated with the
+  Rust project.
+
+# Motivation
+
+Working groups were not created through the RFC process, as such there's not
+much documentation on exactly what a working group does, what are its
+responsibilities, and also importantly setting expectations or goals. There is
+one definition of "Working Group" available in [rust-lang#54445]; quoted below.
+
+> A Rust Working Group is a set of people working at common purpose. Working
+> Groups are associated with a Rust team. Unlike a Rust Team, Working Groups
+> don't have "formal decision making power", though often they are charged with
+> drawing up recommendations, RFCs, or other documents for the teams (which is
+> then intended to make the final decision).
+
+While this definition is true in the broad sense, it does not define the
+difference between what has come to be called "Domain Working Groups" and
+"Team Working Groups". Additionally the community has also adopted the Working
+Group terminology for organising and working on areas they are interested in,
+independent of The Rust Programming Language's Working Groups. It's great that
+we have a community able to self organise in this way, however it has led to
+some confusion over who is supporting these efforts, and whether they're
+considered _"official"_ Working Groups.
+
+This RFC aims to provide clarity by providing new distinct terminology that
+matches the intent behind each of these three separate groups, as well as codify
+the processes that these groups have been using to help facilitate creating
+new groups.
+
+[rust-lang#54445]: https://github.com/rust-lang/rust/issues/54445
+
+# Guide-level explanation
+
+To address this confusion this RFC proposes breaking up the single "Working
+Group" term, into three distinct terms.
+
+- **Working Group** would map most directly to the previous "Domain Working
+  Group" terminology, focused on building and organising around new areas of
+  Rust that aren't covered by the current team roster.
+- **Project Group** would replace the "Team Working Group", and would serve as a
+  catch all term for sub teams for the existing teams to organise around
+  specific efforts, such as certain project or effort in the Rust team.
+- **Community Group** would act as a catch all term for community self
+  organising groups that are independent of the Rust Programming
+  Language Organisation.
+
+## Life-cycle of a Project Group
```

Comment:
Before this section I would add something like this, based on the latest comment on this RFC (please edit as necessary)
--
# A word on shepherd and member time commitments
It is valuable for all Rust contributors to be aware of the time committment involved in joining a project as a member or a shepherd.

It isn't useful to rigorously enforce a time requirement that determines whether a contributor is an active project member.

However, the Python Software Foundation (PSF) provides a useful set of benchmarks for assessing the time committment involved. To be considered a contributor to the PSF, that person should dedicate five hours towards projects that advance the PSF mission, and for someone to be considered a Shepherd (Managing Member), they should dedicate five hours a month towards managing their group

If a project group or working group member finds that they do not have time to work with their team, they can choose to have their name removed from the project during their inactive period and rejoin once they are able to be more active again.
Project and working groups should also be able to discuss the status of potentially inactive members by contacting the relevant individuals directly  . . .
--
I'm writing this a very tired because the email thread drew me in, feel free to tweak. Also, at the  . . . I feel like something more is needed there, but it is a touchy topic so I'm not sure where and how to safely tread . . .

Hope this helps.